Dockerfile

```diff
@@ -1,7 +1,29 @@
 FROM python:3.7-slim-buster
 ENV PYTHONUNBUFFERED 1
-WORKDIR /code
-COPY ./requirements/base.txt /code/
-RUN pip install -r base.txt
-COPY celerylogger/celery_logger.py /code/
-CMD python celery_logger.py
+
+WORKDIR /usr/src/app
+
+# Updating dependencies and installing `gosu`
+RUN set -eux; \
+    apt-get update && \
+    apt-get install -y --no-install-recommends gosu && \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+# Copying package files...
+COPY ./requirements/ /usr/src/app/requirements/
+COPY ./setup.py /usr/src/app/
+COPY ./README.md /usr/src/app/
+COPY celerylogger/ /usr/src/app/celerylogger/
+
+# Creating celerylogger user and adjusting permissions
+RUN set -eux; \
+    groupadd --system --gid=888 celerylogger && \
+    useradd --system --gid=celerylogger --uid=888 \
+            --home-dir=/usr/src/app \
+            --shell /bin/bash celerylogger && \
+    chown -R celerylogger:celerylogger /usr/src/app
+
+# Installing dependencies using the celerylogger user
+RUN gosu celerylogger pip install .
```

Comment:
We probably want to have dependencies installed system wide as root. This way the user running the application won't be able to modify the installed dependencies.

Reply:
Done!